apps/remix-ide/src/app.js

```diff
@@ -405,7 +405,21 @@ Please make a backup of your contracts and start using http://remix.ethereum.org
   // Set workspace after initial activation
   if (Array.isArray(workspace)) {
     try {
```

Comment:
Not sure, if try-catch is still required as we don't have any `await` statement now